arq/connections.py

```diff
@@ -43,8 +44,24 @@ def __repr__(self):
 class ArqRedis(Redis):
     """
     Thin subclass of ``aioredis.Redis`` which adds :func:`arq.connections.enqueue_job`.
+
+    :param redis_settings: an instance of ``arq.connections.RedisSettings``.
+    :param _job_serializer: a function that serializes Python objects to bytes, defaults to pickle.dumps
+    :param _job_deserializer: a function that deserializes bytes into Python objects, defaults to pickle.loads
+    :param kwargs: keyword arguments directly passed to ``aioredis.Redis``.
     """
 
+    def __init__(
+        self,
+        pool_or_conn,
+        _job_serializer: Optional[Callable[[Any], bytes]] = None,
+        _job_deserializer: Optional[Callable[[bytes], Any]] = None,
```

Comment:
```suggestion
        _job_deserializer: Optional[Callable[[bytes], Dict[str, Any]] = None,
```
Or am I missing something?